background.js

```diff
@@ -1,3 +1,5 @@
+var last_time = new Date().valueOf();

```

Comment:
It seems weird to have these as global variables...

Ideally we'd encapsulate the TTS support but at least we should move them to be next to the function that needs them for now.

Reply:
OK. I will move the code to a module.